### ISSUE TITLE
Model checkpoints

### ISSUE THREAD

**Issue by irina-zys** [taneishi/UNet_Glomerulus#1]
Hi,

Thank you for sharing your work in this repository :)

I was wondering if you could share the weights of your final model? Thanks!

**Comment by mosoriob** [comment]
Yes, please. I am working on the same topic and would like to reproduce your code. Thank you.
